treehouses/cli
# treehouses blocker (fixes #657) (fixes #157)

Fixes #657 
Fixes #159 possibly
- Uses https://github.com/StevenBlack/hosts
- ```blocker <0|1|2|3|4|max>```
- Blocks websites through editing of the /etc/hosts file
- To test:
Okay now it works for manual install. Still need to test on new image/npm version. Cheers. 
Run this to test:
```
git clone http://github.com/treehouses/cli
cd cli;
sudo npm i -g --unsafe-perm ./
sudo treehouses blocker
sudo treehouses blocker 1
sudo treehouses blocker
sudo treehouses blocker 3
sudo vim /etc/hosts
```
```/etc/hosts``` should be changing whenever a new blocker level is set.
